pages/api/startHostedActivitySessionViaHostedPagesLink/[hostedPagesLinkId].ts

```diff
@@ -22,14 +22,15 @@ export default async function handler(
   const token = jwt.sign(
     {
       username: environment.apiGatewayConsumerName,
-      feature: 'hosted-pages-link',
+      feature: 'hosted-activities-link',
```

Comment:
same comment as in MR with same change in orchestration. This rename means hosted-pages and awell-next have to be released together